.circleci/config.yml

```diff
@@ -1142,22 +1142,14 @@ jobs:
               }
 
   deploy-to-spm:
-    docker:
-      - image: cimg/base:stable
-    resource_class: small
+    <<: *base-job
```

Comment:
The base job is a Mac machine. Can we use something that's faster/cheaper? Maybe a ruby docker?

Reply:
Done!